torch/csrc/jit/runtime/vararg_functions.h

```diff
@@ -32,7 +32,7 @@ void listConstruct(
 
 void dictConstruct(Stack& stack, const at::DictType& type, size_t num_inputs);
 
-void createObject(Stack& stack, const at::ClassTypePtr& type);
+void createObject(Stack& stack, const at::ClassTypePtr& type, bool as_weak_ref=false);
```

Comment:
let's add a comment describing what the `as_weak_ref` parameter is for, and when you should use it (never)